source/instructors/admin/integration/commoncartridge.rst

```diff
@@ -0,0 +1,22 @@
+.. meta::
+   :description: Export Common Cartride
+
+.. _common-cartridge:
+
+Common Cartridge
+----------------
+
+Allows you to export the `Common Cartridge 1.3 <http://www.imsglobal.org/cc/ccv1p3/imscc_Overview-v1p3.html>`_ data for the course to then use within your LMS system to import details of the assignments in your Codio courses.
+
+1. Navigate to the **Courses** page and select the course to open it.
+2. Click the **LTI/LMS** tab and then in the **LTI/LMS Connections** area, click to **Common Cartridge** the common cartridge information
```

Comment:
Click the **LTI/LMS** tab and then in the **LTI/LMS Connections** area, click to **Common Cartridge**, the common cartridge information.